src/gwt/acesupport/acemode/markdown_highlight_rules.js

```diff
@@ -65,7 +65,7 @@ var escaped = function(ch) {
 function github_embed(tag, prefix) {
     return { // Github style block
         token : "support.function",
-        regex : "^\\s*```" + tag + "\\s*$",
+        regex : "^\\s*```\\{?" + tag + "([^\\}]*\\})?\\s*$",
```

Comment:
Good point! I think I'll just have two different expressions for the plain
and {} cases

On Sun, Apr 10, 2016 at 12:28 PM, Kevin Ushey <notifications@github.com>
wrote:

> In src/gwt/acesupport/acemode/markdown_highlight_rules.js
> <https://github.com/rstudio/rstudio/pull/681#discussion_r59136732>:
>
> > @@ -65,7 +65,7 @@ var escaped = function(ch) {
> >  function github_embed(tag, prefix) {
> >      return { // Github style block
> >          token : "support.function",
> > -        regex : "^\\s*```" + tag + "\\s*$",
> > +        regex : "^\\s*```\\{?" + tag + "([^\\}]*\\})?\\s*$",
>
> I think you need to ensure that only non-capturing regex groups are used
> here.
>
> FWIW I think this also implies that the IDE would accept e.g.
>
> ```{js
> ```js}
>
> when highlighting a block -- is that okay?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/681/files/0f72f6afb500dc7040fa612cbd8a3902c5187ecd#r59136732>
>